charts/jenkins/templates/jenkins-controller-ingress.yaml

```diff
@@ -37,6 +37,7 @@ spec:
             name: {{ template "jenkins.fullname" . }}
             port:
               number: {{ .Values.controller.servicePort }}
+        pathType: ImplementationSpecific
```

Comment:
This is already part of #200

Reply:
it looks like (from my testing), that for `v1beta1`, it defaults to `pathType: ImplementationSpecific`, but for `v1` you need to explicitly set it